### DISCUSSION TITLE
ROMs added but still not recognized

### DISCUSSION THREAD

**Discussion by manveru1986** [PCemOnMac/PCemV17macOS#27] [Q&A] [answered]
I have compiled the most recent version on macOS 11.6 on MBP M1.
Copied the roms, but after running the program, I'm getting the no roms error
Any advices?

**Comment by nlharri** [comment]
I had the same issue, and for me the following information helped a lot:
- https://github.com/PCemOnMac/PCemV17macOS/issues/29#issue-1086598207
- https://github.com/PCemOnMac/PCemV17macOS/issues/29#issuecomment-999657329

The folder I'm using currently for PCem is `~/Library/Application Support/PCem`. This default folder should be documented somewhere.

**Comment by IBMPCXT5150** [comment, marked as answer]
The ROMs should be located in /Users/usr/Library/Application Support/PCem/roms. I Don't mean the /Library/ Directory. I Literally mean that one. There is a hidden Library Folder in your usr folder. You can find it by:
1.Going to your user folder
2.press the two dots to the left of search.
3.Show View Options
4.Enable "Show Library Folder".
